samples/trace/README.md

```diff
@@ -0,0 +1,37 @@
+# Overview
+
+This example shows how to use [@opentelemetry/node](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-node) to instrument a simple Node.js application - e.g. a batch job - and export spans either to [Google Cloud Trace](https://cloud.google.com/trace/).
```

Comment:
Remove "either".
Also, it's generally good style to break long lines (same in multiple places)